index.html.in

```diff
@@ -248,6 +254,20 @@
                                       <p style="display:none" id="fileDownloadBrowserWarning" data-text="Download_Browser_Warning">Your browser might not handle downloads correctly. Either right-click and "Save As" that file-name or open this page in Firefox.</p>
                                   </div>
                               </div>
+                              <div id="translateWebpage" style="display: none; overflow: hidden;">
+                                  <div class="col-sm-12">
+                                      <div class="input-group" style="width: 100%; margin-bottom: 10px;">
+                                        <span class="input-group-addon"><i class="fa fa-link"></i></span>
+                                        <input type="url" id="webpage" placeholder="URL" class="form-control">
+                                      </div>
+                                      <iframe
+                                        id="translatedWebpage"
+                                        class="translatedWebpage"
+                                        src="about:blank"
```

Comment:
This [stackoverflow link](https://stackoverflow.com/questions/5946607/is-an-empty-iframe-src-valid) discusses about how `about:blank` is used to set an empty `iframe`. The comments section reveal that it is widely supported.